### PR TITLE
Integrated ImageEditor in the app for the image editing page. 

### PR DESCRIPTION
**Pull request by Subhankar29**
Fixes issue #446 

Changes: Integrated ImageEditor app's editing activity in the app. The basic layout of the page with the basic functionalities are there.

Screenshots for the change: 
screenshot_2017-05-14-23-01-44-976

### CODE REVIEW

File: imageeditlibrary/build.gradle [hunk 1]
```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 24
+    buildToolsVersion '25.0.0'
```
Comment (pa1pal) [inline imageeditlibrary/build.gradle]: Change the `buildToolsVersion` to 25.0.3 which is mentioned in `travis.yml` to fix the fail checks

File: imageeditlibrary/build.gradle [hunk 2]
```diff
@@ -0,0 +1,26 @@
+apply plugin: 'com.android.library'
+
+android {
+    compileSdkVersion 24
```
Comment (pa1pal) [inline imageeditlibrary/build.gradle]: change it to 25 to match with `travis.yml` to fix the fail checks

**Comment by pa1pal** [comment]
@Subhankar29  resolve the merge conflicts

**Comment by Subhankar29** [comment]
Yes, working on it.

**Comment by pa1pal** [comment]
#464

**Pull request closed without merging**